### DISCUSSION TITLE
Is Material 3 Web even maintained?

### DISCUSSION THREAD

**Discussion by georg-stone** [material-components/material-web#5736] [General]
Material 3 is a beautiful, awesome UI library. The only issue with it is that it's lacking components in its web version. According to the roadmap, React was supposed to come in Q1, more screen readers in Q2, but it's Q4 now. None of this has happened. The web version isn't getting any love at all. My question is: are these new features and components ever gonna be worked on? Or is the Flutter version the only maintained version?

**Comment by asyncliz** [comment]
It is unfortunately in a [minimum maintenance mode](https://github.com/material-components/material-web/discussions/5642) 😞
